components/dashboard/src/settings/Preferences.tsx

```diff
@@ -210,7 +166,7 @@ export default function Preferences() {
                         )}
                         <CheckBox
                             title="Latest Release"
```

Comment:
The language "latest release" does not effectively warn the user of the implications of switching. 

When we say "latest" that sounds like an unequivocally good thing, so begs the question (as a user) "Why would I not check this box?" The naming is also a little misleading, as we call the latest (non EAP and non insiders) versions "latest" also. Alternative options:
1. Latest release (unstable)
1. Latest unstable release
1. Pre release
1. Early release
1. JetBrains Early Access Program(EAP) / VS Code insiders

---

Suggestion...

| Screenshot      | Copy |
| ----------- | ----------- |
| ![image](https://user-images.githubusercontent.com/5528307/160824970-45c1a09c-2629-4a71-bcd3-6c1e292ec8dd.png) | > **Enable VS Code Insiders / JetBrains Early Access Program (EAP)** Use the latest available IDE or editor version for early access to new features, however stability is not guaranteed. |

It would also be good to include links to both schemes for users who are not familiar (we could later update these with links to our own documentation):
1. [VS Code Insiders](https://code.visualstudio.com/blogs/2016/02/01/introducing_insiders_build) 
2. [JetBrains Early Access Program (EAP)](https://www.jetbrains.com/resources/eap/)

Reply:
> It would also be good to include links to both schemes for users who are not familiar

+1

To clarify, the _Latest Release_ term here was used for providing brevity and making it less complex or scary for users opt in. Using a short term like this can also help make alerts like the one introduced in this PR in the workspace start page to be less verbose.

However, I agree we could make this slightly better and I'm slightly in favor of the first option @loujaybee provided above, see also relevant comment in https://github.com/gitpod-io/gitpod/issues/7932#issuecomment-1026672526. 

| Checkbox for latest release (unstable) |
|-|
| <img width="1152" alt="Screenshot 2022-03-30 at 12 53 07 PM" src="https://user-images.githubusercontent.com/120486/160828532-a2f1452e-7bea-4de5-a218-bcd7f62aa971.png"> |